wot-wg-2023-draft.html

```diff
@@ -81,28 +81,29 @@ <h1 id="title">PROPOSED Web of Things Working Group Charter</h1>
     <!-- delete PROPOSED after AC review completed -->
 
     <p class="mission">
-      The <strong>mission</strong> of the
-      <a href="https://www.w3.org/groups/wg/wot">Web of Things Working Group</a>
-      is to counter the fragmentation of the IoT through
-      the specification of building blocks
-      that enable easy integration of IoT devices and services
-      across IoT platforms and application domains.
-      These building blocks should complement and enhance existing standards.
-      This
-      Working Group Charter
-      covers those aspects that the
-      <a href="http://www.w3.org/groups/ig/wot">Web of Things Interest Group</a>
-      <a href="https://www.w3.org/2021/12/wot-ig-2021.html">(charter here)</a>
-      believes are mature enough to progress to W3C Recommendations.
+      <!-- counter the fragmentation needs to be changed but not sure what to use -->
+      The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/wot">Web of Things Working Group</a>
+      is to counter the fragmentation of the IoT through the specification of building blocks
+      that enable easy integration of IoT devices and services across IoT platforms and application domains.
+      These building blocks should complement and enhance the use of existing standards, and allow WoT to
+      be used within other ecosystems and communities.
+      This reduces the initial friction for adoption and supports further industry adoption.
+      <span class="todo">This Working Group Charter covers those aspects that the
+        <a href="http://www.w3.org/groups/ig/wot">Web of Things Interest Group</a>
+        <a href="https://www.w3.org/2021/12/wot-ig-2021.html">(charter here)</a>
+        believes are mature enough to progress to W3C Recommendations.
+      </span>
+      <!-- This is not true anymore and can be removed. IG does not write any reports. All reports, even informative ones are done by the WG.
```

Comment:
Use case and requirements document is an IG report, so....

Reply:
This is true, I am rewording